runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -25,7 +31,113 @@ namespace train
 namespace ops
 {
 
-PoolLayer::PoolLayer() : cpu::ops::PoolLayer()
+namespace
+{
+
+cpu::ops::PoolType convert_to_infer(const train::ops::PoolType &pool_type)
```

Comment:
(Optional)
```suggestion
cpu::ops::PoolType convertToInfer(const train::ops::PoolType &pool_type)
```
Although there are many places in onert that did not follow naming conventions, it would be better to use camel case that starts with lowercase.

Reply:
updated